frontend/latest/package.json

```diff
@@ -1,8 +1,17 @@
 {
-  "//": "This is also plugin code, and must snake case (since it's used as federation module name",
+  "//": "This is also plugin code, and must snake case (since it's used as federation module name), and should be unique for all plugins",
   "name": "plugin_examples",
-  "//": "Must be set for plugin generation CLI to consider this file as plugin",
-  "isOpenMsupplyPlugin": true,
+  "//": "When plugins are bundled in production, this field of package.json is used to identify this directory",
+  "//": "As a frontend plugins, unlike backend plugins 'types' here is just metadata, only used to tell user which area plugin affects",
+  "omSupplyPlugin": {
```

Comment:
Since bundling of plugins is done by examining package.json and both front end and back end plugins use package.json as definition of the plugin, now using this structure